cps/kobo.py

```diff
@@ -392,6 +394,222 @@ def get_metadata(book):
     return metadata
 
```

Comment:
for the early PR: line 382 onwards:
```
        if sys.version_info < (3, 0):
            name = get_series(book).encode("utf-8")
        else:
            name = get_series(book)
        metadata["Series"] = {
            "Name": get_series(book),
```
looks suspicious to me to insert unencoded series and only use the decoded text to generate the id, could you please doublecheck it (I didn't do it up to now)

Reply:
also following line:
```
       metadata["Series"] = {
            "Name": get_series(book),
            "Number": book.series_index,        # ToDo Check int() ?
            "NumberFloat": float(book.series_index),
```
From earlier commit. Could you please check: The book.series_index is a "float" like string with up to 2 decimals (meanwhile) shouldn't the books.series_index be casted to  int() ?